demo/vagrant/Vagrantfile

```diff
@@ -50,6 +50,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.define name="rook%02d" % i, primary: (i == 0), autostart: (i == 0) do |node|
       node.vm.hostname = name
       node.vm.network "private_network", ip: "172.20.20.#{10+i}"
+      node.vm.network "forwarded_port", guest: 8124, host: 8124
```

Comment:
yes, that maybe should have been a separate commit, but it's way easy to test from your ubuntu host now with the rook binary from the local bin tree.  very happy with this simple change :)